queries/systemverilog/highlights.scm

```diff
@@ -0,0 +1,562 @@
+; Keywords
+[
+  "begin"
+  "end"
+  "generate"
+  "endgenerate"
+  (module_keyword)
+  "endmodule"
+  "program"
+  "endprogram"
+  "package"
+  "endpackage"
+  "checker"
+  "endchecker"
+  "config"
+  "endconfig"
+  "pure"
+  "virtual"
+  "extends"
+  "implements"
+  "super"
+  (class_item_qualifier)
+  "parameter"
+  "localparam"
+  "defparam"
+  "assign"
+  "typedef"
+  "modport"
+  "fork"
+  "join"
+  "join_none"
+  "join_any"
+  "default"
+  "break"
+  "tagged"
+  "extern"
+  "alias"
+  "posedge"
+  "negedge"
+  "bind"
+  "expect"
+  "type"
+  "void"
+  "coverpoint"
+  "cross"
+  "nettype"
+  "export"
+  "force"
+  "release"
+  "timeunit"
+  "timeprecision"
+  "sequence"
+  "endsequence"
+  "property"
+  "endproperty"
+  "clocking"
+  "endclocking"
+  "covergroup"
+  "endgroup"
+  "specify"
+  "endspecify"
+  "primitive"
+  "endprimitive"
+  "wait"
+  "wait_order"
+  "const"
+  "constraint"
+  "unique"
+  "do"
+  "genvar"
+  "inside"
+  "rand"
+  "continue"
+  "randc"
+  "event"
+  "global"
+  "ref"
+  (unique_priority)
+  (bins_keyword)
+  (always_keyword)
+] @keyword
+
+[
+  "class"
+  "endclass"
+  "interface"
+  "endinterface"
+  "enum"
+  "struct"
+  "union"
+] @keyword.type
+
+[
+  "function"
+  "endfunction"
+  "task"
+  "endtask"
+] @keyword.function
+
+"return" @keyword.return
+
+[
+  "for"
+  "foreach"
+  "repeat"
+  "forever"
+  "initial"
+  "while"
+] @keyword.repeat
+
+[
+  "if"
+  "else"
+  "iff"
+  (case_keyword)
+  "endcase"
+] @keyword.conditional
+
+[
+  "="
+  "-"
+  "+"
+  "/"
+  "*"
+  "^"
+  "&"
+  "|"
+  "&&"
+  "||"
+  "<="
+  "=="
+  "!="
+  "==="
+  "!=="
+  "-:"
+  "<"
+  ">"
+  ">="
+  "%"
+  ">>"
+  "<<"
+  "|="
+  "|=>"
+  "|->"
+  ">>>"
+  "<<<"
+  "->>"
+  "->"
+  "=>"
+  "*>"
+  ".*"
+  "?"
+  ":"
+  "#"
+  (unary_operator)
+  (inc_or_dec_operator)
+  (queue_dimension)
+] @operator
+
+[
+  ";"
+  "::"
+  ","
+  "."
+] @punctuation.delimiter
+
+[
+  "["
+  "]"
+  "("
+  ")"
+  "{"
+  "}"
+  "'{"
+] @punctuation.bracket
+
+[
+  "or"
+  "and"
+] @keyword.operator
+
+[
+  "input"
+  "output"
+  "inout"
+  "signed"
+  "unsigned"
+  "assert"
+  "cover"
+  "assume"
+  "disable"
+  "automatic"
+  "static"
+  (dpi_function_import_property)
+  (dpi_task_import_property)
+] @keyword.modifier
+
+[
+  "include"
+  "import"
+] @keyword.import
+
+(comment) @comment @spell
+
+[
+  "@"
+  (cycle_delay_range)
+  (delay_control)
+  (cycle_delay)
+  (timescale_compiler_directive)
+  (time_literal)
+  (attribute_instance)
+] @attribute
+
+(attribute_instance
+  (attr_spec
+    (simple_identifier) @property))
+
+[
+  (integral_number)
+  (unbased_unsized_literal)
+  (fixed_point_number)
+  (unsigned_number)
+] @number
+
+[
+  (net_type)
+  (data_type)
+  (timescale_compiler_directive)
```

Comment:
This node is also captured earlier as `@attribute`. I think both these nodes should be removed, with `"directive_timescale"` captured as `@keyword.directive`, and `(time_unit)` as `@type.builtin`